### PR TITLE
[DELTA-35] first version config file

### PR DESCRIPTION
**Pull request by deltafilzr**
ready for review, since wormhole configuration for the chain is mainly dependent on the wormhole chain ID, right now just keep name and id as fields for chain parameter

### CODE REVIEW

File: src/tokenBridge/fullDeployConfig.json [hunk 1]
```diff
@@ -0,0 +1,320 @@
+{
+  "solana-config": {
```
Comment (pjxthy) [inline src/tokenBridge/fullDeployConfig.json:2]: solana-config -> solana since -config is redundant.

File: src/tokenBridge/fullDeployConfig.json [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+  "chain": {
```
Comment (pjxthy) [inline src/tokenBridge/fullDeployConfig.json:3]: add indent to show the hierarchical structure

File: src/tokenBridge/fullDeployConfig.json [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+    "wormholeChainId" : "CHAIN_ID_SOLANA",
+    "name" : "Solana"
+  },
+  "mainnet-prod": {
```
Comment (pjxthy) [inline src/tokenBridge/fullDeployConfig.json:7]: remove all solana config and keep polygon and bnb configs only.

File: src/tokenBridge/fullDeployConfig.json [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+        "name": "DeltaFi Token",
+        "mint": "4iefnqUWWcn8nWTyXdyHqYkLwn1UPov18vCyP9T8Fy68",
+        "symbol": "DELFI",
+        "decimals": 6,
+        "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/de1QJkP1qDCk5JYCCXCeq27bQQUdCaiv7xVKFrhPSzF/logo.png"
+      }
+    ]
+  }
+  },
+  "ethereum-config": {
```
Comment (pjxthy) [inline src/tokenBridge/fullDeployConfig.json:305]: for token list, let us keep bnb and wrapped bnb on polygon only for now.

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/tokenBridge/fullDeployConfig.json (hunk 1 above)
Comment (deltafilzr) [reply src/tokenBridge/fullDeployConfig.json:2]: ok

**Pull request closed without merging**